src/main/java/org/candlepin/subscriptions/tally/admin/InternalTallyResource.java

```diff
@@ -89,6 +100,9 @@ public InternalTallyResource(
     this.properties = properties;
     this.eventRecordRepository = eventRecordRepository;
     this.eventRecordsRetentionProperties = eventRecordsRetentionProperties;
+    this.eventKafkaTemplate = eventKafkaTemplate;
+    this.objectMapper = objectMapper;
+    this.eventTopic = "platform.rhsm-subscriptions.service-instance-ingress";
```

Comment:
This value should come from the properties. See `SERVICE_INSTANCE_INGRESS_TOPIC`.